com.carpeludum.KegaFusion.json

```diff
@@ -0,0 +1,86 @@
+{
+    "app-id" : "com.carpeludum.KegaFusion",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
```

Comment:
Update runtime to 19.08

Reply:
The first comment literally explains why I can't do that.

Incidentally, it's starting to get seriously annoying seeing your only interactions with me and the software that I maintain be "upgrade to this runtime version" with no explanations as to what it would entail, not thinking about the work it would take, and not answering when I ask those.